### PR TITLE
changed Open to IsOpen

### PR DESCRIPTION
**Pull request by Monaldcry7788**

### CODE REVIEW

**Comment by joker-119** [comment]
I'm not sure if this change is really necessary?

**Comment by NaoUnderscore** [comment]
I'd prefer using IsOpen, also it's better to change it now to avoid breaking everything in future, like in the Exiled Beta.

File: Exiled.API/Features/Door.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public Door(DoorVariant door)
         /// <summary>
         /// Gets or sets a value indicating whether the door is open.
         /// </summary>
-        public bool Open
```
Comment (PintTheDragon) [inline Exiled.API/Features/Door.cs:56]: Please add the `Open` property back, but mark it as Obsolete.

**Review by PintTheDragon — APPROVED** [review]

File: Exiled.API/Features/Door.cs (hunk 1 above)
Comment (Monaldcry7788) [reply Exiled.API/Features/Door.cs:56]: Ok, give me a few minutes

Comment (joker-119) [reply Exiled.API/Features/Door.cs:56]: That isn't necessary during the alpha stage @PintTheDragon

**Review by PintTheDragon — APPROVED** [review]
My first approval was meant to be a change request. I'm actually approving it this time.

**Pull request merged**